README.md

```diff
@@ -47,8 +47,18 @@ Opcard uses [virtualsmartcard](https://frankmorgner.github.io/vsmartcard/) for t
 `make dangerous-real-card-test` will instead run the tests against a real card.
 The vendor id and serial numbers can be configured with variables:
```

Comment:
Please mention that it also tries to start and stop pcscd.  This could be surprising and cause problems, depending on your setup.